Makefile

```diff
@@ -197,7 +197,7 @@ bazel-build: bazel-fetch
 	$(BAZEL) $(BAZEL_OPTS) build $(BAZEL_BUILD_OPTS) $(BUILD_TARGET)
 
 bdist_wheel: bazel-build
-	$(PYTHON) setup.py bdist_wheel
+	$(PYTHON) setup.py bdist_wheel --package-dir bazel-bin/package.runfiles/CompilerGym
```

Comment:
Really minor nitpick: no need to set this flag as it is the default value.

Reply:
Done.